bee-message/src/payload/milestone/option/mod.rs

```diff
@@ -0,0 +1,230 @@
+// Copyright 2022 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+mod pow;
+mod receipt;
+
+use derive_more::{Deref, From};
+use iterator_sorted::is_unique_sorted;
+use packable::{bounded::BoundedU8, prefix::BoxedSlicePrefix, Packable};
+pub(crate) use receipt::{MigratedFundsAmount, ReceiptFundsCount};
+
+pub use self::{
+    pow::PowMilestoneOption,
+    receipt::{MigratedFundsEntry, ReceiptMilestoneOption, TailTransactionHash},
+};
+use crate::Error;
+
+///
```

Comment:
Docs

Reply:
Yes I know, will be done in another PR 👍